src/sanity/schemaTypes/documents/site.ts

```
@@ -12,149 +12,122 @@ import { VscGlobe } from 'react-icons/vsc';
     }),
   ],
   preview: {
-    prepare: () => ({
-      title: 'Site settings',
+    select: {
+      title: 'title',
+      tagline: 'tagline',
+    },
+    prepare: ({ title, tagline }) => ({
+      title: title || 'Site settings',
+      subtitle: tagline && tagline[0]?.children ? tagline[0].children.map((c: any) => c.text).join(' ') : '',
```

Biome (1.9.4)

[error] Change to an optional chain.

Unsafe fix: Change to an optional chain.


(lint/complexity/useOptionalChain)